Assignment3_bash.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+wget https://downloads.mariadb.com/Connectors/java/connector-java-2.4.3/mariadb-java-client-2.4.3.jar
```

Comment:
I did not run your code, as the MariaDB connector is not working on my system. But I think the logic you have put in, actually makes sense. So everything should work fine.

Reply:
I just thought it would be easier for anyone running the code to just download mariadb connector and add to the jars file. But now i am having doubts of whether i should keep it as everyone has a different python versions too

Reply:
I think it is for the best if you keep it, at least that way you can tell everyone what works on your system. I would suggest you can comment about it, that way people can understand that this may or may not work on their PC's so they can have an alternative of using the MYSQL connector.